pkg/controller/operconfig/migration.go

```diff
@@ -516,8 +541,8 @@ func netNamespaceHasEgressIpConfig(nns uns.Unstructured) bool {
 }
 
 func hostSubnetHasEgressIpConfigAutomatic(hsn uns.Unstructured) bool {
-	if hsn.Object["egressIPs"] != nil && hsn.Object["egressCIDRs"] != nil {
-		return len(hsn.Object["egressIPs"].([]interface{})) > 0 && len(hsn.Object["egressCIDRs"].([]interface{})) > 0
+	if hsn.Object["egressCIDRs"] != nil {
```

Comment:
can you explain why the changes here?

Reply:
The customer may configure more nodes that can host egress IP (by setting `egressCIDRs` to hostsubnets) than the number of egress IP. So we shouldn't assume that all these nodes have an egress IP allocated. We shall just label those nodes with `egress-assignable` while migrating the configuration.